steps/cli/oc.go

```diff
@@ -165,6 +166,16 @@ var _ = gauge.Step("Update addon config with resolverStepActions as <resolverSte
 	}
 })
 
+var _ = gauge.Step("Verify test versioned ecosystem task", func() {
+	taskList := cmd.MustSucceed("oc", "get", "task", "-n", "openshift-pipelines").Stdout()
+	requriedTasks := []string{"git-cli", "git-clone", "s2i-go", "s2i-python"}
```

Comment:
1. typo
2. extend this list to all tasks

Reply:
Done